spec/element_hidden_spec.rb

```diff
@@ -0,0 +1,65 @@
+require File.expand_path('watirspec/spec_helper', File.dirname(__FILE__))
+
+describe Watir::Locators::Element::Locator do
```

Comment:
Yup but again, Travis won't show you if they pass if they are in watirspec. :) I planned to move before merging.